src/amber/environment/settings.cr

```diff
@@ -10,6 +10,27 @@ module Amber::Environment
       skip: Array(String?),
       context: Array(String?))
 
+
+    struct SmtpSettings
```

Comment:
Should we treat struct naming the same as class naming?
> Acronyms
> In class names, acronyms are all-uppercase. For example, HTTP, and LibXML.
> In method names, acronyms are all-lowercase. For example #from_json, #to_io.

Source: https://crystal-lang.org/docs/conventions/coding_style.html

If so, it should be `SMTPSettings`, right?

Reply:
👍